### PR TITLE
Feat: Alert Component

### PR DESCRIPTION
**Pull request by SaurabhChawda**

### CODE REVIEW

File: assets/Css/ComponentsStyles.css [hunk 1]
```diff
@@ -212,3 +212,55 @@ main {
...
+}
+.alert__title{
+  font-size:1rem;
+  font-weight: 900;
+  padding:0.5rem;
+}
+
+/* Alert Color */
+.alert--success{
+  background-color: rgb(0, 199, 0);
```
Comment (shivampandey0) [inline assets/Css/ComponentsStyles.css:252]: use variables instead of hard coded color values

**Review by shivampandey0 — COMMENTED** [review]
The background color looks too much blue 😅

File: Components/Alert.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        </ul>
+    </div>
+    <!-- Navigation Bar For Tab and Mobile  -->
+    <div class="nav-bar nav-bar--hamburger">
+        <div class="nav-bar--logo">
+            <a href="/index.html"><img class="nav-bar__img--logo" src="/assets/images/AQUA_UI Logo.png"
+                    alt="AQUA_UI"></a>
+        </div>
+        <div class="nav-bar--hamburger">
+            <img class="nav-bar__img--hamburger" src="/assets/images/Hamburger Menu.png" alt="">
```
Comment (barbaricankit) [inline Components/Alert.html:33]: always give a value to the alt.

File: assets/Css/ComponentsStyles.css (hunk 1 above)
Comment (SaurabhChawda) [reply assets/Css/ComponentsStyles.css:252]: okay, I will convert it into a variable.
ThankYou

File: Components/Alert.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                </a>
+                <a href="">
+                    <li class="sidebar__item">Toast</li>
+                </a>
+            </ul>
+        </div>
+
+        <!-- Alert -->
+        <!-- Alert Intro -->
+        <div class="alert card">
```
Comment (barbaricankit) [inline Components/Alert.html:119]: you will be creating a card component so is it good to use a class name as card over here?

**Comment by ShreyashMahajan** [comment]
everything looks good as well as it is responsive also

File: Components/Alert.html [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+            <div class="card-demo alert__intro">
+                <h1 class="card-demo--title alert__intro--title">Alert</h1>
+                <p class="card-demo--desc alert__intro--desc"><b>Alert</b> is used for Quick Actions and user feedback.
```
Comment (barbaricankit) [inline Components/Alert.html:122]: you could have used info, just a suggestion

**Comment by SaurabhChawda** [comment]
> everything looks good as well as it is responsive also

ThankYou

File: assets/Css/ComponentsStyles.css [hunk 5]
```diff
@@ -212,3 +212,55 @@ main {
...
+}
+.alert__img{
+  width: 1.5rem;
+  height: 1.5rem;
+}
+.alert__text{
+  font-size:0.8rem;
+  margin-left:0.5rem;
+}
+/* Alert Modified */
```
Comment (barbaricankit) [inline assets/Css/ComponentsStyles.css:232]: What is alert modified?

**Review by barbaricankit — COMMENTED** [review]
The class names are not proper enough, Also for each component have a separate CSS file so that in the future it is easy to debug or make any change.

**Pull request merged**